src/main/java/cinema/shop/service/impl/MovieSessionServiceImpl.java

```
@@ -25,13 +26,22 @@ public MovieSession add(MovieSession movieSession) {
         return movieSessionDao.add(movieSession);
     }
 
+    @Override
+    public MovieSession get(Long id) {
+        Optional<MovieSession> optionalMovieSession = movieSessionDao.get(id);
+        if (optionalMovieSession.isEmpty()) {
+            return null;
```

let's throw an exception instead